browser/ui/views/tabs/brave_tab_group_underline.cc

```diff
@@ -5,18 +5,29 @@
 
 #include "brave/browser/ui/views/tabs/brave_tab_group_underline.h"
 
+#include <algorithm>
+
+#include "brave/browser/ui/views/tabs/brave_tab_group_header.h"
 #include "brave/browser/ui/views/tabs/features.h"
+#include "cc/paint/paint_flags.h"
 #include "chrome/browser/ui/views/tabs/tab_group_views.h"
+#include "ui/gfx/canvas.h"
+#include "ui/gfx/geometry/skia_conversions.h"
+
+BraveTabGroupUnderline::BraveTabGroupUnderline(
+    TabGroupViews* tab_group_views,
+    const tab_groups::TabGroupId& group)
+    : TabGroupUnderline(tab_group_views, group) {
+  SetEnabled(false);
```

Comment:
please add comment explaining why we set this disabled by default. Also shouldn't this be guarded by vertical tabs feature flag?